drivers/w1/w1_ds2477_85_common.h

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2022 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_W1_DS2477_DS2485_COMMON_H_
+#define ZEPHYR_DRIVERS_W1_DS2477_DS2485_COMMON_H_
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/w1.h>
+#include <zephyr/kernel.h>
+
+/* memory specific commands */
+#define CMD_WRITE_MEM		 0x96
+#define CMD_READ_MEM		 0x44
+#define CMD_SET_PAGE_PROTECT	 0xc3
+#define CMD_READ_STATUS		 0xAA
+/* configuration specific commands */
+#define CMD_SET_I2C_ADDR	 0x75
+#define CMD_READ_W1_PORT_CONFIG	 0x52
+#define CMD_WRITE_W1_PORT_CONFIG 0x99
+#define CMD_MASTER_RESET	 0x62
+/* 1-Wire specific commands */
+#define CMD_W1_SCRIPT		 0x88
+#define CMD_W1_BLOCK		 0xAB
+#define CMD_READ_BLOCK		 0x50
+#define CMD_WRITE_BLOCK		 0x68
+#define CMD_SEARCH		 0x11
+#define CMD_FULL_CMD_SEQ	 0x57
+/* crc16 specific commands */
+#define CMD_COMPUTE_CRC		 0xCC
+
+/* I2C communication result bytes */
+#define RESULT_SUCCESS	     0xAA
```

Comment:
Would be better if these are namespaced so that they wont conflict with other stuff

Reply:
Prefixed these results bytes with, DS24.
Such that a conflict is unlikely, but the names are still not too long (i.e. DS24_RES_SUCCESS)